cirq-google/cirq_google/experimental/noise_models/noise_models.py

```diff
@@ -96,6 +114,21 @@ def noisy_moment(self, moment: cirq.Moment, system_qubits: Sequence[cirq.Qid]) -
                     )
             return moments
 
+    def _json_dict_(self) -> dict[str, object]:
+        return {
+            'depol_probs': tuple((q, p) for q, p in (self.depol_probs or {}).items()),
+            'bitflip_probs': tuple((q, p) for q, p in (self.bitflip_probs or {}).items()),
+            'decay_probs': tuple((q, p) for q, p in (self.decay_probs or {}).items()),
```

Comment:
will it work with returning the dictionaries as they are?
Also, consider using [obj_to_dict_helper](https://github.com/quantumlib/Cirq/blob/6bb666165e7bb0265e8ebdc408a2fb09274c4005/cirq-core/cirq/protocols/json_serialization.py#L137-L151) here.